src/AccessibilityInsights.Extensions.AzureDevOps/AzureDevOpsIntegration.cs

```diff
@@ -486,6 +487,29 @@ internal Uri CreateIssuePreview(string projectName, string teamName, IReadOnlyDi
             Uri.TryCreate(finalUrl, UriKind.Absolute, out Uri result);
             return result;
         }
+
+        /// <summary>
+        /// Retains characters within ascii range 1-127 and returns
+        /// an escaped representation of the string
+        /// </summary>
+        /// <param name="str"></param>
+        /// <returns></returns>
+        internal static string EscapeForUrl(string str)
+        {
+            var sb = new StringBuilder();
+            foreach (var c in str)
+            {
+                // characters such as the middle dot (ascii 183) can 
+                // cause issues during navigation, so we only include
+                // a basic set of characters when building the URL
+                if (c >= 1 && c <= 127)
```

Comment:
>c >= 1 [](start = 20, length = 6)

Do we need the lower bounds check? I'd expect it to be unsigned and maybe even support null